src/shaii/Shaii.shaii.ts

```diff
@@ -318,7 +325,9 @@ class Shaii {
             clearTyping();
 
             // This is pretty cringe
-            if (error == "TypeError: Cannot read property 'getUniformLocation' of null") {
+            try {
+              new WebGLRenderer();

```

Comment:
no we don't want to try making a new webglrenderer everytime someone does a command

Reply:
in terms of performance the other way is better 💀

Reply:
Well yeah I see
But there is probably a cleaner way to check it tho 💀